src/frontend/src/handler/show.rs

```diff
@@ -106,6 +108,17 @@ fn schema_or_default(schema: &Option<Ident>) -> String {
         .map_or_else(|| DEFAULT_SCHEMA_NAME.to_string(), |s| s.real_value())
 }
 
+fn schema_or_search_path<'a>(
+    schema: &'a Option<Ident>,
+    search_path: &'a SearchPath,
+) -> impl Iterator<Item = String> + 'a {
```

Comment:
This does not seem to compile.

Reply:
Yes, just a preview of whether the pr can resolve the issue.